core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/exception/ClusterNotFoundException.java

```diff
@@ -12,13 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.expediagroup.streamplatform.streamregistry.exceptions;
+package com.expediagroup.streamplatform.streamregistry.core.exception;
```

Comment:
FWIW... the whole exception tree needs to be rethought IMO

Reply:
Yep. I removed the unused ones. Would be great to get your feedback on the next PR :)

Reply:
Yup!